test/docfx.RegressionTest/RegressionTest.cs

```diff
@@ -181,19 +183,15 @@ private static string EnsureTestData(Options opts, string workingFolder, int ret
                     Directory.CreateDirectory(workingFolder);
                     Exec("git", $"init", cwd: workingFolder);
                     Exec("git", $"remote add origin {TestDataRepositoryUrl}", cwd: workingFolder);
-                    Exec("git", $"{s_gitCmdAuth} fetch origin --progress template", cwd: workingFolder, secrets: s_gitCmdAuth);
+
+                    Retry().Execute(() =>
```

Comment:
It seems all git network operations needs retry, consider a `ExecGitRetry(cmd, ...)` method because the name `Retry()` does not tell that it only checks for git failures and retry.

Reply:
Ok